### PR TITLE
bpftool: add new recipe

### PR DESCRIPTION
**Pull request by mrchapp**

### CODE REVIEW

File: recipes-kernel/bpftool/bpftool.bb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+do_configure[depends] += "virtual/kernel:do_shared_workdir"
+
+do_compile() {
+    oe_runmake
+}
+
+do_install() {
+    oe_runmake DESTDIR=${D} install
+    chown -R root:root ${D}
```
Comment (fboudra) [inline recipes-kernel/bpftool/bpftool.bb:23]: the chown seems fishy. Maybe a comment will help to understand why it is required.

**Review by fboudra — COMMENTED** [review]
I don't think it's the right place for the recipe. Since it has been submitted to meta-oe, I think it's best to submit to meta-backports. meta-96boards is a BSP layer.

File: recipes-kernel/bpftool/bpftool.bb (hunk 1 above)
Comment (mrchapp) [reply recipes-kernel/bpftool/bpftool.bb:23]: Do we need it? I'm not getting any warnings without chown.

I got that from cpupower, but here it's not making a difference on hackbox.

Comment (mrchapp) [reply recipes-kernel/bpftool/bpftool.bb:23]: Doesn't seem to be required for bpftool.

**Comment by mrchapp** [comment]
Sent to meta-backports: https://review.linaro.org/26164

**Pull request closed without merging**